rest_api/pipelines.yaml

```diff
@@ -0,0 +1,45 @@
+version: '0.7'
```

Comment:
Don't seeing anywhere we are checking version

Reply:
The plan is to implement in the next iterations. It's still useful to have it for documentation purprose or issues to be able to correlate with the Haystack version.